components/homepage/NewsFeed.js

```diff
@@ -76,7 +76,7 @@ class NewsFeed extends React.Component {
               <br />
               <a href="/data">Explore the Data</a>
               <br />
-              <a href="/contact">Get Involved</a>
+              <a href="/volunteer">Get Involved</a>
```

Comment:
What do you think about using `<Link>` here to take advantage of Next.js's [client-side routing](https://nextjs.org/docs/api-reference/next/link)? I think we could make a similar change for the other local links on this page.

```suggestion
              <Link href="/volunteer"><a>Get Involved</a></Link>
```